src/network/core/packet.cpp

```diff
@@ -37,9 +38,13 @@ Packet::Packet(NetworkSocketHandler *cs, size_t initial_read_size) : next(nullpt
 
 /**
  * Creates a packet to send
- * @param type of the packet to send
+ * @param type  The type of the packet to send
+ * @param limit The maximum number of bytes the packet may have. Default is COMPAT_MTU.
+ *              Be careful of compatibility with older clients/servers when changing
+ *              the limit as it might break things if the other side is not expecting
+ *              much larger packets than what they support.
  */
-Packet::Packet(PacketType type) : next(nullptr), pos(0), cs(nullptr)
+Packet::Packet(PacketType type, size_t limit) : next(nullptr), pos(0), cs(nullptr)
```

Comment:
Bit late, I know, but Packet::limit doesn't seem to be being assigned here.

Reply:
Thanks to LordAro it's fixed now. Thanks for spotting it.
I'm fairly certain it got lost when solving a conflict during one of the many rebases I had, so I guess I need to improve that process somehow to prevent changes to get lost in rebases.